src/pretix/static/pretixpresale/js/ui/main.js

```diff
@@ -513,24 +513,24 @@ $(function () {
 
     $("input[data-required-if], select[data-required-if], textarea[data-required-if]").each(function () {
         var dependent = $(this),
-            dependentLabel = $("label[for="+this.id+"]"),
-            dependency = $($(this).attr("data-required-if")),
+            dependencies = $($(this).attr("data-required-if")),
             update = function (ev) {
-                var enabled = (dependency.attr("type") === 'checkbox' || dependency.attr("type") === 'radio') ? dependency.prop('checked') : !!dependency.val();
-                if (!dependent.is("[data-no-required-attr]")) {
-                    dependent.prop('required', enabled);
-                }
```

Comment:
removing this will break the name_parts widget in this configuration:
(it will make optional name parts required)


<img width="782" height="362" alt="image" src="https://github.com/user-attachments/assets/9830f5cf-858d-4556-ad68-d2e4616689c1" />



<img width="782" height="362" alt="image" src="https://github.com/user-attachments/assets/e89d4437-9852-45c0-ac9a-318b0c108b70" />